### PR TITLE
Upgrade Source v3.0.1

### PR DESCRIPTION
**Pull request by Source-Protocol-Cosmos**
Small Security Update. Organized with Validators via Discord. All Validators currently running v3.0.1.

### CODE REVIEW

**Comment by reversesigh** [comment]
Is this a prop based upgrade or a patch for the current version? I couldn't find a prop for this on ping pub.

**Comment by Source-Protocol-Cosmos** [comment]
Patch to the current version. No Gov Prop.

**Comment by Source-Protocol-Cosmos** [comment]
Updated

File: source/chain.json
Comment (reversesigh) [inline source/chain.json:1]: The changes to v3.0.0 are correct, but you still need to remove lines 65-78 for v3.0.1 then we can merge.

Comment (Source-Protocol-Cosmos) [reply source/chain.json:1]: Done

**Pull request merged**